core/templates/pages/admin-page/admin-page.directive.ts

```diff
@@ -49,12 +49,12 @@ angular.module('oppia').directive('adminPage', [
         '/pages/admin-page/admin-page.directive.html'),
       controllerAs: '$ctrl',
       controller: [
-        '$location', '$rootScope', '$scope', 'AdminDataService',
```

Comment:
Do we need to remove AdminDataService import/require from the top?

Reply:
It is only used for userEmail here and the getDataAsync gives AdminPageData which doesn't have userEmail.